figma-screenshots/MentorStatusDesigns/README.md

```diff
@@ -0,0 +1,19 @@
+# Mentor Status - Setting Availability
+Detailed and displayed here is the design for [BL 709 - As a Mentor I want to set my availability](https://bloomtechlabs.atlassian.net/jira/software/c/projects/BL/boards/10?modal=detail&selectedIssue=BL-709&quickFilter=27&quickFilter=19)
+
+The BE route that hits the DS API to toggle the Mentor's `accepting_new_mentees` boolean is in the process of being merged. After discussions with appropriate parties we have decided to add this feature to the nav bar for simplicity and findability.
```

Comment:
Well done Jon! This sort of functionality is proof of the project's dedication to user-synchronization and your personal efforts to make things easier for the rest of your team.  Keep it up!!!

Reply:
Much appreciated!